docs/conceptual/runtime/frame.md

```diff
@@ -1,29 +1,26 @@
 ---
-title: Substrate Runtime Module Library
+title: Framework for Runtime Aggregation of Modularised Entities
 ---
 
-The Substrate Runtime Module Library (SRML) is a collection of prebuilt runtime modules and support
-libraries which simplify development of a Substrate runtime.
+The Framework for Runtime Aggregation of Modularised Entities (FRAME) is a collection of prebuilt pallets and support libraries which simplify development of a Substrate runtime.
```

Comment:
line length (I believe preferred is 100)

(also s -> z in this paragraph)

Reply:
Also Joe provided some fodder about the framework itself beyond the collection of pallets. I quoted it [here](https://github.com/substrate-developer-hub/substrate-developer-hub.github.io/issues/324#issuecomment-565089750)

> FRAME is a set of modules (called _pallets_) and support libraries that simplify runtime development. Pallets are individual modules within FRAME that host domain-specific logic.
>
> FRAME provides some helper modules to interact with Substrate Primitives, namely Executive, Metadata, Support, System, and Utility. Any module that uses these `frame-*` helper modules is a pallet. Parity develops and maintains
>a set of these, but they can come from the community as well.
>
>If you don't want to use the FRAME helpers, you can still write your own interfaces to interact with the Substrate client and implement your custom runtime.